client/my-sites/plans/jetpack-plans/storage-tier-upgrade/hooks/use-get-storage-upgrade-product.ts

```diff
@@ -0,0 +1,108 @@
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import slugToSelectorProduct from 'calypso/my-sites/plans/jetpack-plans/slug-to-selector-product';
+import { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+import { TIER_1_SLUGS, TIER_2_SLUGS } from '../constants';
+
+const useGetTier1UpgradeProduct = () => {
+	const translate = useTranslate();
+
+	return useCallback(
+		( slug: string ): SelectorProduct =>
+			( {
+				...slugToSelectorProduct( slug ),
+				displayName: translate( '20GB' ),
+				subheader: translate( 'of backup storage' ),
+				buttonLabel: translate( 'Upgrade storage' ),
+				// description: <the default description of the product being upgraded>,
+				features: {
+					items: [
+						{
+							text: translate( '20GB backup storage' ),
+							isHighlighted: true,
+						},
+						{
+							text: translate( 'One-click restore from the last 30 days of backups' ),
+							isHighlighted: true,
+						},
+						{
+							text: translate( '30-day activity log' ),
+							isHighlighted: true,
+						},
+						{
+							text: translate( 'Real-time backups (as you edit)' ),
+						},
+						{
+							text: translate( 'Backups are cloud based and secure' ),
+						},
+					],
+				},
+			} as SelectorProduct ),
+		[ translate ]
+	);
+};
+
+const useGetTier2UpgradeProduct = () => {
+	const translate = useTranslate();
+
+	return useCallback(
+		( slug: string ): SelectorProduct =>
+			( {
+				...slugToSelectorProduct( slug ),
+				displayName: translate( '1TB' ),
+				subheader: translate( 'of backup storage' ),
+				buttonLabel: translate( 'Upgrade storage' ),
+				description: translate(
+					'Go back in time and recover all your information for up to a year, with 1TB storage space.'
+				),
+				features: {
+					items: [
+						{
+							text: translate( '1TB backup storage' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Backup (1TB)', { context: 'Jetpack Backup Solution'} )` *ES Score: 6*